components/molecule/modal/src/Content/index.js

```diff
@@ -0,0 +1,51 @@
+import {useRef} from 'react'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+
+const MoleculeModalContent = ({
+  className: classNameProp,
+  enableContentScroll,
+  withoutIndentation,
+  ...others
+}) => {
+  const baseClassName = 'sui-MoleculeModalContent'
+  const contentRef = useRef()
+  const className = cx(baseClassName, {
+    [`${baseClassName}--without-indentation`]: withoutIndentation
```

Comment:
Following SUITCSS naming convention, modifier should be in camelCase: https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md
```suggestion
    [`${baseClassName}--withoutIndentation`]: withoutIndentation
```